### PR TITLE
AEROGEAR-3292 - material icon updates

### PR DESCRIPTION
**Pull request by jasonmadigan**
Before:
screen shot 2018-06-14 at 15 42 43

After:
screen shot 2018-06-14 at 15 42 09

### CODE REVIEW

**Comment by jasonmadigan** [comment]
landing page updated also

Old:
screen shot 2018-06-14 at 15 53 39

New:
screen shot 2018-06-14 at 15 53 54

File: _includes/sidebar-services.html [hunk 1]
```diff
@@ -1,20 +1,32 @@
-<ul class="nav">
-    <li>
-        <a href="/services/mobile-ci-cd/"><i class="fa fa-cogs" aria-hidden="true"></i><strong> CI/CD</strong></a>
+<ul class="nav services-sidebar">
+    <li> 
+      <a href="/services/metrics/">
+        <i class="material-icons" aria-hidden="true">insert_chart</i> Metrics
```
Comment (dimitraz) [inline _includes/sidebar-services.html]: I know this isn't part of this PR, but would it be possible to change the order of these to match the order in the navbar?

**Comment by jasonmadigan** [comment]
Updated from review comments. Screenshot if new PWA & React icons below:

screen shot 2018-06-18 at 15 59 47

File: _includes/sidebar-news.html [hunk 2]
```diff
@@ -97,6 +97,22 @@ <h5>Platforms:</h5>
   </span>
 </div>
 
+<div class="checkbox platform">
```
Comment (laurafitzgerald) [inline _includes/sidebar-news.html]: in this page can we also add the custom connector to the module section?
```
 <div class="checkbox module">
   <span class="label label-primary">
   <a href="#module-custom-runtime-connector">
      <i class="fa fa-plug"></i> Custom Runtime Connector
     </a>
   </span>
 </div>
```

Comment (laurafitzgerald) [reply _includes/sidebar-news.html]: and metrics 
```
<div class="checkbox module">
  <span class="label label-primary">
    <a href="#module-metrics">
      <i class="fa fa-bar-chart"></i> Metrics
    </a>
  </span>
</div>
```

Comment (laurafitzgerald) [reply _includes/sidebar-news.html]: also the push icon should be updated.

**Pull request merged**